secret/secrets_manager_client_test.go

```diff
@@ -61,11 +72,77 @@ func TestSecretsManagerCreateAndDeleteSecret(t *testing.T) {
 		require.NoError(t, err)
 		require.NotZero(t, out)
 
+		defer Cleanup(out)
+
+		require.NotZero(t, out.Name)
+		require.NotZero(t, out.ARN)
+
+	})
+
+	t.Run("GetFailsWithInvalidInput", func(t *testing.T) {
+		out, err := c.GetSecretValue(ctx, &secretsmanager.GetSecretValueInput{})
+		assert.Error(t, err)
+		assert.Zero(t, out)
+	})
+
+	t.Run("UpdateFailsWithInvalidInput", func(t *testing.T) {
+		out, err := c.UpdateSecret(ctx, &secretsmanager.UpdateSecretInput{})
+		assert.Error(t, err)
+		assert.Zero(t, out)
+	})
+
+	t.Run("CreateAndGetSucceed", func(t *testing.T) {
+		out, err := c.CreateSecret(ctx, &secretsmanager.CreateSecretInput{
+			Name:         aws.String(os.Getenv("AWS_SECRET_PREFIX") + "foobar"),
+			SecretString: aws.String("barfoo"),
```

Comment:
Can we use random strings for this too?

Reply:
Do you mean the Name, SecretString, or both?

Reply:
Just the name, since it has to be unique. The secret value can be whatever you want.